build.gradle

```diff
@@ -117,7 +117,7 @@ configurations {
 }
 
 ext.odata_version = '4.2.0'
-ext.chronicle_api_version = '0.0.12'
+ext.chronicle_api_version = '0.0.13'

```

Comment:
This one doesn't exist yet.

Reply:
but it will be when this is merged to master right? So I don't think there is any harm bumping up now.